### PR TITLE
Change declared and concluded licenses to relationships

### PR DESCRIPTION
**Pull request by goneall**
Implements decision made on whether licenses should be a property or relationship on the 2023-05-02 tech call

Note that this introduces issues as documented in #254

This fixes the parsing errors #443

Also fixes #Signed-off-by: Gary O'Neall <gary@sourceauditor.com>

### CODE REVIEW

**Comment by goneall** [comment]
Per the [minutes from 2023-05-02](2023-05-02) @kestewart @tsteenbe and @swinslow are to review the pull request for changing from properties to relationships.

**Comment by goneall** [comment]
Note: I would like to merge this to fix the parser errors.

File: model/Core/Vocabularies/RelationshipType.md [hunk 1]
```diff
@@ -25,10 +25,12 @@ between a Package and a File, between two Packages, or between one SPDXDocument
 - buildDependency: Every `to` Element is a build dependency of the `from` Element
 - buildTool: Build tool used to build an Element. This may be used to describe the build tool of a Build instance
 - coordinatedBy: (Security) Used to identify the vendor, researcher, or consumer agent performing coordination for a vulnerability
+- concludedLicense: Identifies the license that that SPDX data creator has concluded as governing the software Package, File or Snippet.
```
Comment (zvr) [inline model/Core/Vocabularies/RelationshipType.md:28]: ... "governing the software Artifact."

File: model/Core/Vocabularies/RelationshipType.md [hunk 2]
```diff
@@ -25,10 +25,12 @@ between a Package and a File, between two Packages, or between one SPDXDocument
...
 - contains: Every `to` Element is contained by the `from` Element
 - configOf: (Build) Configuration information applied to an Element instance during a LifeycleScopeType period.  Example: Build configuration of the build instance
 - copy: Every `to` Element is a copy of the `from` Element
 - dataFile: Every `to` Element is a data file related to the the `from` Element
+- declaredLicense: dentifies the license information actually found in the software Package, File or Snippet, for example as detected by use of automated tooling.
```
Comment (zvr) [inline model/Core/Vocabularies/RelationshipType.md:33]: "software Artifact"

**Review by zvr — CHANGES_REQUESTED** [review]

File: model/Core/Vocabularies/RelationshipType.md (hunk 2 above)
Comment (kestewart) [reply model/Core/Vocabularies/RelationshipType.md:33]: aslo please fix typo "dentifies" --> "Identifies",   can you also be explict as to the "to" and "from".  ie.  what goes on which side of the relationship.

File: model/Core/Vocabularies/RelationshipType.md (hunk 1 above)
Comment (kestewart) [reply model/Core/Vocabularies/RelationshipType.md:28]: Please be explicit, what is "to" and what is "from" here

**Review by kestewart — CHANGES_REQUESTED** [review]
Please be explicit about what is "to" and what should be "from" for concludedLicense & declaredLicense.

File: model/Core/Vocabularies/RelationshipType.md (hunk 2 above)
Comment (goneall) [reply model/Core/Vocabularies/RelationshipType.md]: Updated per comments

**Review by kestewart — APPROVED** [review]
Thanks.   That fixes the issue note.

**Comment by goneall** [comment]
Since @zvr  is away from the office this week and I have made his requested changes plus we have a positive review from @kestewart , I'm going to merge this.  It should fix the error's we're seeing in the CI.

If anyone has any additional improvements on this, please open a new PR.

**Pull request merged**